### PR TITLE
Added missing fr translations

### PR DESCRIPTION
**Pull request by Cali0707**
**Description**

This adds in the missing FR translations ahead of the April release. There is only one remaining `MISSING` tag in the french translations, and that is because there is a corresponding `MISSING` tag in the english translations.

**Type of change**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Translations update

**How Has This Been Tested?**

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Passes test case
- [ ] UI components visually reviewed on desktop view
- [ ] UI components visually reviewed on mobile view
- [ ] Other (please explain)

**Checklist:**

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] The precommit and linting ran successfully
- [ ] I have added or updated language tags for text that's part of the UI
- [ ] I have added "MISSING" for all new language tags to languages I don't speak
- [ ] I have added [the GNU General Public License](https://lite-farm.atlassian.net/l/cp/BT0Dd7WW) to all new files

### CODE REVIEW

**Comment by Duncan-Brain** [comment]
Thanks for this. Just wondering if the other files in the /fr/ folder and other translation locations like in the app are also meant to be part of this?

**Comment by Cali0707** [comment]
> @Cali0707 Thanks for this. Just wondering if the other files in the /fr/ folder and other translation locations like in the app are also meant to be part of this?

@Duncan-Brain good catch, I'd forgotten to check the other files - they should have the missing translations addressed now

File: packages/webapp/public/locales/fr/translation.json [hunk 1]
```diff
@@ -958,7 +958,7 @@
     "TITLE": "Inviter un utilisateur",
     "WAGE": "Salaire",
     "WAGE_ERROR": "Le salaire doit être un nombre décimal valide et non négatif",
-    "WAGE_RANGE_ERROR": "MISSING"
+    "WAGE_RANGE_ERROR": "Le salaire doit être un nombre positif inférieur à 999 999 999 999"
```
Comment (SayakaOno) [inline packages/webapp/public/locales/fr/translation.json:961]: It looks like there are three extra 9s..?
English: "Salary must be a positive number less than 999,999,999"

File: packages/webapp/public/locales/fr/translation.json [hunk 2]
```diff
@@ -1712,102 +1712,99 @@
...
+    "TITLE": "Vous avez choisi un emplacement biologique!",
+    "SUBTITLE": "Vous avez dit que vous allez planter une culture biologique dans un endroit non biologique. Voulez-vous commencer?"
   },
   "WAGE": {
     "ERROR": "Le salaire doit être un nombre décimal valide et non négatif",
-    "HOURLY_WAGE": "MISSING",
-    "HOURLY_WAGE_RANGE_ERROR": "MISSING",
-    "HOURLY_WAGE_TOOLTIP": "MISSING"
+    "HOURLY_WAGE": "Salaire horaire",
+    "HOURLY_WAGE_RANGE_ERROR": "Le salaire horaire doit être un nombre positif inférieur à 999 999 999 999",
```
Comment (SayakaOno) [inline packages/webapp/public/locales/fr/translation.json:1807]: It looks like there are three extra 9s..?
English: "Hourly Wage must be a positive number less than 999,999,999"

**Review by SayakaOno — COMMENTED** [review]
Thank you @Cali0707!
It would be great if you could double-check the numbers.

**Comment by SayakaOno** [comment]
@Duncan-Brain 
I don't think we show a soil water potential graph, so it's okay if `SOIL_WATER_POTENTIAL_READINGS_OF_SENSOR.SUBTITLE` is "MISSING". Carolina was also wondering, could you confirm?

Screenshot 2023-04-06 at 10 17 14 AM

File: packages/webapp/public/locales/fr/translation.json (hunk 1 above)
Comment (Cali0707) [reply packages/webapp/public/locales/fr/translation.json]: Whoops, my bad - should be fixed now.

File: packages/webapp/public/locales/fr/translation.json (hunk 2 above)
Comment (Cali0707) [reply packages/webapp/public/locales/fr/translation.json]: Should also be fixed now

**Comment by Duncan-Brain** [comment]
@SayakaOno 

Sorry I missed this, yes we show soil water potential graphs.

**Comment by SayakaOno** [comment]
@Duncan-Brain so that means we need to add subtitle? Would you mind creating a ticket? I can't provide how one should add data to show a soil water potential graph. (I can make a ticket if there's no need to test on the UI)

**Review by SayakaOno — APPROVED** [review]

**Pull request merged**